tests/staticfiles_tests/test_finders.py

```diff
@@ -106,13 +106,17 @@ def test_searched_locations(self):
         )
 
     @override_settings(STATICFILES_DIRS='a string')
-    def test_non_tuple_raises_exception(self):
+    def test_non_tuple_does_not_raise_improperly_configured(self):
         """
-        We can't determine if STATICFILES_DIRS is set correctly just by
-        looking at the type, but we can determine if it's definitely wrong.
+        Regression test (#27661).
         """
-        with self.assertRaises(ImproperlyConfigured):
+        try:
             finders.FileSystemFinder()
+        except ImproperlyConfigured:
+            self.fail('FileSystemFinder should not raise ImproperlyConfigured anymore. '
```

Comment:
I think that word `anymore` is unnecessary. `'FileSystemFinder should not raise ImproperlyConfigured.` sounds better to me.